core/parachain/pvf/pvf_worker_types.hpp

```diff
@@ -39,9 +39,9 @@ namespace kagome::parachain {
     bool force_disable_secure_mode;
   };
 
-  using PvfWorkerInputCode = std::string;
+  using RuntimeCodePath = std::string;
```

Comment:
```suggestion
  using PvfWorkerInputCodePath = std::string;
```

Reply:
I don't like it, but since you both agree, then okay.